soc/riscv/riscv-privilege/telink_b9x/soc.c

```diff
@@ -315,14 +327,51 @@ void soc_b9x_restore(void)
 	}
 }
 
-static inline uint32_t read_flash_mid(void)
+#if CONFIG_SOC_RISCV_TELINK_B95
+
+#include "flash/flash_common.h"
+#include "lib/include/flash_base.h"
```

Comment:
"lib/include" can be removed

Reply:
will add the lib/include into cmake list .